src/components/Posts/ExpandedPostReplies.tsx

```diff
@@ -25,13 +26,17 @@ const ExpandedPostReplies = ({ post }: ExpandedPostRepliesProps) => {
             },
           }
         );
-        dispatch(setPosts(resultReplies.data as Post[]));
+        const modified = resultReplies.data.map((entry: any) => ({
+          ...entry,
+          timestamp: formatTimestamp(entry),
+        }));
+        dispatch(setPosts(modified as Post[]));
       } catch (e) {
         console.log(e.message);
       }
     };
     fetchPosts();
-  }, [dispatch, user, post]);
+  }, [dispatch, user, post, posts]);
```

Comment:
I think we should revert this back to how it was originally because adding `posts` to the dependency array seems to infinitely call the `useEffect` for some reason. Try `console.log` in the useEffect and tell me if you get the same result